decidim-admin/app/controllers/decidim/admin/scopes_controller.rb

```diff
@@ -74,9 +88,33 @@ def scope
         @scope ||= collection.find(params[:id])
       end
 
+      def parent_scope
```

Comment:
I've changed the core to allow browsing the scopes tree with the same controller and views. So, I need to know if I'm showing a top level scope or a child. In the second case, this method gives me, the parent scope.